src/v2/controllers/FormController.js

```diff
@@ -141,9 +147,12 @@ export default Controller.extend({
 
     if (idx['neededToProceed'].find(item => item.name === actionPath)) {
       idx.proceed(actionPath, {})
-        .then(this.handleIdxSuccess.bind(this))
+        .then(this.handleIdxResponse.bind(this))
```

Comment:
if Idx responded with either success or error.. for widget it doesn't matter if it was a success or failure. Previously, API errors were not formatted as idx responses.. https://github.com/okta/okta-idx-js/pull/66/files